### ISSUE TITLE
Keyboard accessibility issue and aria values

### ISSUE THREAD

**Issue by equinusocio** [mhayes/vue-twentytwenty#5]
Hi, we have some issue making this component WCAG compliant because it's not accessible by keyboard.

**Comment by mhayes** [comment]
Hey @equinusocio I'm not very familiar with WCAG and what kind of functionality would need to be added to this plugin to make it keyboard accessible. If you have some time to develop a PR I'd be more than happy to take a look.

**Comment by equinusocio** [comment]
@mhayes Done.
